App/Config/sentinelConfig.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+/**
+ * Part of the Sentinel package.
+ *
+ * NOTICE OF LICENSE
+ *
+ * Licensed under the 3-clause BSD License.
+ *
+ * This source file is subject to the 3-clause BSD License that is
+ * bundled with this package in the LICENSE file.
+ *
+ * @package    Sentinel
+ * @version    2.0.13
+ * @author     Cartalyst LLC
+ * @license    BSD License (3-clause)
+ * @copyright  (c) 2011-2016, Cartalyst LLC
+ * @link       http://cartalyst.com
+ */
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Session Key
+    |--------------------------------------------------------------------------
+    |
+    | Please provide your session key for Sentinel.
+    |
+    */
+
+    'session' => 'cartalyst_sentinel',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cookie Key
+    |--------------------------------------------------------------------------
+    |
+    | Please provide your cookie key for Sentinel.
+    |
+    */
+
+    'cookie' => 'cartalyst_sentinel',
+
+    /*
+    |--------------------------------------------------------------------------
+    | Users
+    |--------------------------------------------------------------------------
+    |
+    | Please provide the user model used in Sentinel.
+    |
+    */
+
+    'users' => [
+
+        'model' => \App\Models\UsersModel::class,
+
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Roles
+    |--------------------------------------------------------------------------
+    |
+    | Please provide the role model used in Sentinel.
+    |
+    */
+
+    'roles' => [
+
+        'model' => 'Cartalyst\Sentinel\Roles\EloquentRole',
```

Comment:
Lets go ahead and be consistent, use the php static class method reference on `Cartalyst\Sentinel\Roles\EloquentRole`.

```php
'model' => \Cartalyst\Sentinel\Roles\EloquentRole::class,
```